Package.swift

```diff
@@ -10,14 +10,16 @@ let package = Package(
         .library(name: "SourceryJS", targets: ["SourceryJS"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/kylef/Commander.git", from: "0.6.1"),
```

Comment:
I am seeing `0.6.0` for Commander in [Podfile.lock](https://github.com/krzysztofzablocki/Sourcery/blob/56b999afa87272bb9c70b907ff4e1922723ba91c/Podfile.lock#L3).